accelerator/config.c

```diff
@@ -14,13 +14,24 @@
 
 static logger_id_t logger_id;
 
-int get_conf_key(char const* const key) {
-  for (int i = 0; i < cli_cmd_num; ++i) {
+/**
+ * @brief Get corresponding key-value pair in ta_cli_arguments_g structure
```

Comment:
Should it be "Get the corresponding value of the key"?

Reply:
let me change it